src/bootstrap.tsx

```diff
@@ -48,6 +60,23 @@ const App = () => {
   return <RootApp cookieElement={cookieElement} setCookieElement={setCookieElement} />;
 };
 
+const ConfigLoader = () => {
+  const initPreview = useSetAtom(isPreviewAtom);
+  function initSuccess(userConfig: ChromeUserConfig) {
+    initPreview(userConfig.data.uiPreview);
+  }
+  function initFail() {
+    initPreview(false);
```

Comment:
I wonder if it's worth displaying a notification in this scenario so it's clear we've reverted back to non-preview due to a config init fault.